api/client/clientRouter.js

```diff
@@ -80,10 +80,12 @@ router.put(':id', (req, res) => {
 
 /**
  * @swagger
- * /:id/notes
+ * /{id}/notes:
  *  get:
```

Comment:
What did you switch from using a `:` to using curly brackets? 
Did this have to do with the JSON formatting correctly?

Reply:
Yes, this is the correct way to format YAML.